docs/examples/en/animations/CCDIKSolver.html

```diff
@@ -92,7 +92,12 @@ <h3>[method:CCDIKHelper createHelper]()</h3>
 
 		<h3>[method:CCDIKSolver update]()</h3>
 		<p>
-		Update bones quaternion by solving CCD algorithm.
+		Update IK bones quaternion by solving CCD algorithm.
+		</p>
+
+		<h3>[method:CCDIKSolver update]( [param:Object ikParam] )</h3>
+		<p>
+		Update an IK bone quaternion by solving CCD algorithm.
```

Comment:
This doesn't seem correct?

Reply:
Oops, the new one should be `updateOne`. I'll fix, thanks.

Reply:
Updated, thanks!